pkg/agent/proxy/proxier.go

```diff
@@ -470,25 +482,33 @@ func (p *proxier) installServices() {
 				klog.ErrorS(err, "Error when installing Endpoints flows")
 				continue
 			}
+			var exportedSvcGroupID binding.GroupIDType
```

Comment:
How about using a pointer here? Then you also don't need to use a dummy variable like `zeroGroupID`.

Reply:
Another question: for a mc-cluster Service, is it possible has more than one Endpoints which are all Services, if so, I think we need a slice to collect these exported Service group IDs, not a variable.

Reply:
> I don't get your question, for any exported Service, it will have only one corresponding local Service's ClusterIP as Endpoint of MC Service(IPv4 only).

Is it possible that a mc Service has multiple local Services as Endpoints?

Reply:
No, it's 1-1 mapping.